start-here/js/script-main.js

```diff
@@ -78,3 +78,8 @@ optionsList.forEach((option) => {
   }
   option.addEventListener("click", handler);
 });
+// Dark Mode Toggle
+function myFunction() {
+  var element = document.body;
+  element.classList.toggle("dark-mode");
+ }
```

Comment:
I think you need to add an event listener for the checkbox and connect it to this function.

Reply:
@CBID2  need help with it?

Reply:
> @CBID2  need help with it?

Hey @Iroh-Omolola, I haven't been feeling well for the past week, so I can definitely use your help.